src/App.css

```diff
@@ -46,6 +46,47 @@ body{
   }
 }
 
+.react-switch-checkbox {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.react-switch-label {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  width: 60px;
+  height: 30px;
+  background: rgb(247, 186, 186);
+  border-radius: 100px;
+  position: relative;
+  transition: background-color .2s;
+}
+
+.react-switch-label .react-switch-button {
+  content: '';
+  position: absolute;
+  top: 2px;
+  left: 2px;
+  width: 25px;
```

Comment:
I recommend using `margin` and `padding` properties for creating label styles here instead of setting `width`. Setting width works, but it makes it more difficult and tedious to make your layouts responsive at different view widths.

Reply:
This post also does a good job of illustrating why you should avoid setting fixed heights and widths of certain elements.

https://ashleemboyer.com/blog/don-t-use-fixed-css-height-or-width-on-text-containers